guided_contribution_tutorial.md

```diff
@@ -0,0 +1,35 @@
+# Guided contribution tutorial
+
+DALI is an open-source project, therefore we are more than happy to accept external contributions (refer to [CONTRIBUTION.md](https://github.com/NVIDIA/DALI/blob/main/CONTRIBUTING.md) for details about regular contributions). Moreover, we are also keen on closer and more structured cooperation. For instance, you can work on a more significant contribution to DALI as a part of your school project, Deep Learning oriented university class, Bachelor's or Master thesis etc... We also recommend such project to be implemented by a group of developers, so if you have any friends who would like to join you, that's even better. This guide outlines the whole process, from scratch to your first merged PR.
+
+1. If you are ready to commit more time to develop DALI, please contact us directly. You can use either Github issues or write to us on [via email](mailto:dali-team@nvidia.com). We will agree upon the contribution goal, make the plan, sign the necessary NDA (Non-disclosure agreement) and establish a more direct communication channel.
+1. I'm assuming, that we have already set up some communication channel (e.g. Slack, mail, etc…). If we didn't yet, please ask your contact point in our team to do so.
+1. You have been assigned with a DALI-team developer, who would help you with everything you need. Let's call him The Mentor. That shall be your primary contact point with us. Nevertheless, feel free to reach out to anyone from the team, we all are happy to help.
```

Comment:
Everything starts with `1.`

Reply:
Yes, it does ;)

Please look at the rendered file